code/game/objects/items/shields.dm

```diff
@@ -172,20 +196,7 @@
 	var/repair_material = /obj/item/stack/sheet/mineral/titanium
 	var/can_shatter = TRUE
 	shield_flags = SHIELD_FLAGS_DEFAULT | SHIELD_TRANSPARENT
-	max_integrity = 75
-
-/obj/item/shield/run_block(mob/living/owner, atom/object, damage, attack_text, attack_type, armour_penetration, mob/attacker, def_zone, final_block_chance, list/block_return)
-	if(ismovable(object))
-		var/atom/movable/AM = object
-		if(CHECK_BITFIELD(shield_flags, SHIELD_TRANSPARENT) && (AM.pass_flags & PASSGLASS))
-			return BLOCK_NONE
-	if(attack_type & ATTACK_TYPE_THROWN)
-		final_block_chance += 30
-	if(attack_type & ATTACK_TYPE_TACKLE)
-		final_block_chance = 100
-	. = ..()
-	if(. & BLOCK_SUCCESS)
-		on_shield_block(owner, object, damage, attack_text, attack_type, armour_penetration, attacker, def_zone, final_block_chance, block_return)
+	max_integrity = 450
```

Comment:
i'm not sure i should be buffing it that much but the reasoning behind buffing integrity is: 

shields currently deduct the real damage of something, not blocked or post-blocked damage.
so say you block 200 lasers
logically you take 10 0damage and the shield takes 100
right now no, the shield takes 200 damage.